assets/templates/partials/gtm-data-layer.tmpl

```diff
@@ -14,12 +25,16 @@
         }
 
         // ons_cookie_policy handler
-        var policyCookie = document.cookie.match(/^(.*)?\s*'usage':true\s*[^;]+(.*)?$/);
+        var policyCookie = document.cookie.match('(?:^|; )ons_cookie_policy=({.*?})');
         if (policyCookie) {
-            console.debug('ons_cookie_policy found - opting in');
+            console.debug('ons_cookie_policy found');
+
+            var usageValue = extractValue("usage", policyCookie[1]);
+            console.debug('usage is', usageValue);
+
             // this needs to be the inverse - if usage is true the returned value is false and vice versa
             // user is stating whether they are opting out of usage cookie
-            return false
+            return !usageValue
```

Comment:
This is not being interpreted by the `getUsageCookieValue()` function as expected

ie. user has the usage as `false` the opt out settings should be `true` (the inverse)

<img width="369" alt="Screenshot 2025-05-13 at 09 32 05" src="https://github.com/user-attachments/assets/dd0a6a5d-2b26-4a8a-9b7f-6d1626953a2a" />